app/controllers/dor_controller.rb

```diff
@@ -34,14 +36,24 @@ def delete_from_index
 
   private
 
+  def cocina_hash
+    @cocina_hash ||=
+      if params[:cocina_object].present?
+        # params[:cocina_object] is itself an ActionController::Parameters instance, #permit! is needed to allow the #to_h
+        params[:cocina_object].permit!.to_h
```

Comment:
```suggestion
        params[:cocina_object].to_unsafe_h
```

Reply:
did this manually with your other suggestion and amended the most recent commit w/ both changes